docs/release/trg-8/trg-8-05.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: TRG 2.06 - Dependabot
+title: TRG 8.05 - Dependabot
 ---
 
 | Status | Created      | Post-History    |
 |--------|--------------|-----------------|
+| Update | 19-July-2024 | Moved the content from trg-2-6.md|
```

Comment:
please keep the table formatting consistent

Reply:
HI @ndr-brt: I have made the necessay corrections. Please verify.